--- Python-2.7.18/Modules/timemodule.c.orig
+++ Python-2.7.18/Modules/timemodule.c
@@ -806,10 +806,17 @@
 #if defined(HAVE_TZNAME) && !defined(__GLIBC__) && !defined(__CYGWIN__)
     tzset();
 #ifdef PYOS_OS2
     PyModule_AddIntConstant(m, "timezone", _timezone);
 #else /* !PYOS_OS2 */
+#if defined(MS_WINDOWS) && defined(_MSC_VER) && _MSC_VER >= 1900
+    /* Visual Studio 2015+ requires underscore-prefixed versions */
+    _tzset();
+    PyModule_AddIntConstant(m, "timezone", _timezone);
+    PyModule_AddIntConstant(m, "altzone", _timezone-3600);
+    PyModule_AddIntConstant(m, "daylight", _daylight);
+    PyModule_AddObject(m, "tzname", Py_BuildValue("(zz)", _tzname[0], _tzname[1]));
+#else
     PyModule_AddIntConstant(m, "timezone", timezone);
-#endif /* PYOS_OS2 */
 #ifdef HAVE_ALTZONE
     PyModule_AddIntConstant(m, "altzone", altzone);
 #else
@@ -820,6 +827,8 @@
     PyModule_AddIntConstant(m, "daylight", daylight);
     PyModule_AddObject(m, "tzname",
                        Py_BuildValue("(zz)", tzname[0], tzname[1]));
+#endif
+#endif /* PYOS_OS2 */
 #else /* !HAVE_TZNAME || __GLIBC__ || __CYGWIN__*/
 #ifdef HAVE_STRUCT_TM_TM_ZONE
     {
