diff -ur Python-2.7.18.orig/PCbuild/_bsddb.vcxproj Python-2.7.18/PCbuild/_bsddb.vcxproj
--- Python-2.7.18.orig/PCbuild/_bsddb.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/_bsddb.vcxproj	2025-12-10 21:59:50
@@ -67,7 +67,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1e180000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e180000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/_ctypes.vcxproj Python-2.7.18/PCbuild/_ctypes.vcxproj
--- Python-2.7.18.orig/PCbuild/_ctypes.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/_ctypes.vcxproj	2025-12-10 21:59:50
@@ -64,7 +64,7 @@
       <AdditionalIncludeDirectories>..\Modules\_ctypes\libffi_msvc;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1D1A0000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D1A0000</BaseAddress>
       <AdditionalOptions>/EXPORT:DllGetClassObject,PRIVATE /EXPORT:DllCanUnloadNow,PRIVATE %(AdditionalOptions)</AdditionalOptions>
     </Link>
   </ItemDefinitionGroup>
diff -ur Python-2.7.18.orig/PCbuild/_elementtree.vcxproj Python-2.7.18/PCbuild/_elementtree.vcxproj
--- Python-2.7.18.orig/PCbuild/_elementtree.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/_elementtree.vcxproj	2025-12-10 21:59:50
@@ -65,7 +65,7 @@
       <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;USE_PYEXPAT_CAPI;XML_STATIC;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1D100000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D100000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/_msi.vcxproj Python-2.7.18/PCbuild/_msi.vcxproj
--- Python-2.7.18.orig/PCbuild/_msi.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/_msi.vcxproj	2025-12-10 21:59:50
@@ -62,7 +62,7 @@
   <ItemDefinitionGroup>
     <Link>
       <AdditionalDependencies>cabinet.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1D160000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D160000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/_multiprocessing.vcxproj Python-2.7.18/PCbuild/_multiprocessing.vcxproj
--- Python-2.7.18.orig/PCbuild/_multiprocessing.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/_multiprocessing.vcxproj	2025-12-10 21:59:50
@@ -62,7 +62,7 @@
   <ItemDefinitionGroup>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1e1D0000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e1D0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/_socket.vcxproj Python-2.7.18/PCbuild/_socket.vcxproj
--- Python-2.7.18.orig/PCbuild/_socket.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/_socket.vcxproj	2025-12-10 21:59:50
@@ -62,7 +62,7 @@
   <ItemDefinitionGroup>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1e1D0000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e1D0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/_sqlite3.vcxproj Python-2.7.18/PCbuild/_sqlite3.vcxproj
--- Python-2.7.18.orig/PCbuild/_sqlite3.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/_sqlite3.vcxproj	2025-12-10 21:59:50
@@ -65,7 +65,7 @@
       <PreprocessorDefinitions>MODULE_NAME="sqlite3";%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1e180000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e180000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/_testcapi.vcxproj Python-2.7.18/PCbuild/_testcapi.vcxproj
--- Python-2.7.18.orig/PCbuild/_testcapi.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/_testcapi.vcxproj	2025-12-10 21:59:50
@@ -62,7 +62,7 @@
   </PropertyGroup>
   <ItemDefinitionGroup>
     <Link>
-      <BaseAddress>0x1e1F0000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e1F0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/bz2.vcxproj Python-2.7.18/PCbuild/bz2.vcxproj
--- Python-2.7.18.orig/PCbuild/bz2.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/bz2.vcxproj	2025-12-10 21:59:50
@@ -67,7 +67,7 @@
       <PreprocessorDefinitions>WIN32;_FILE_OFFSET_BITS=64;_CRT_SECURE_NO_DEPRECATE;_CRT_NONSTDC_NO_DEPRECATE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1D170000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D170000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/python.vcxproj Python-2.7.18/PCbuild/python.vcxproj
--- Python-2.7.18.orig/PCbuild/python.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/python.vcxproj	2025-12-10 21:59:50
@@ -66,7 +66,7 @@
     <Link>
       <SubSystem>Console</SubSystem>
       <StackReserveSize>2000000</StackReserveSize>
-      <BaseAddress>0x1d000000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1d000000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/pythoncore.vcxproj Python-2.7.18/PCbuild/pythoncore.vcxproj
--- Python-2.7.18.orig/PCbuild/pythoncore.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/pythoncore.vcxproj	2025-12-10 21:59:50
@@ -74,7 +74,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1e000000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e000000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/pythonw.vcxproj Python-2.7.18/PCbuild/pythonw.vcxproj
--- Python-2.7.18.orig/PCbuild/pythonw.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/pythonw.vcxproj	2025-12-10 21:59:50
@@ -65,7 +65,7 @@
   <ItemDefinitionGroup>
     <Link>
       <StackReserveSize>2000000</StackReserveSize>
-      <BaseAddress>0x1d000000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1d000000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/select.vcxproj Python-2.7.18/PCbuild/select.vcxproj
--- Python-2.7.18.orig/PCbuild/select.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/select.vcxproj	2025-12-10 21:59:50
@@ -61,7 +61,7 @@
   <ItemDefinitionGroup>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1D110000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D110000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
diff -ur Python-2.7.18.orig/PCbuild/unicodedata.vcxproj Python-2.7.18/PCbuild/unicodedata.vcxproj
--- Python-2.7.18.orig/PCbuild/unicodedata.vcxproj	2020-04-19 23:13:39
+++ Python-2.7.18/PCbuild/unicodedata.vcxproj	2025-12-10 21:59:50
@@ -61,7 +61,7 @@
   </PropertyGroup>
   <ItemDefinitionGroup>
     <Link>
-      <BaseAddress>0x1D120000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D120000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
