--- Python-2.7.18-orig/PCbuild/_tkinter.vcxproj	2025-12-12 20:03:35.113187800 +0100
+++ Python-2.7.18-mod/PCbuild/_tkinter.vcxproj	2025-12-12 21:54:39.111611300 +0100
@@ -62,6 +62,10 @@
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <CharacterSet>NotSet</CharacterSet>
   </PropertyGroup>
+  <!-- On ARM64, disable _tkinter as tcl/tk ARM64 support is not yet configured -->
+  <PropertyGroup Condition="'$(Platform)'=='ARM64'" Label="Configuration">
+    <ConfigurationType>Utility</ConfigurationType>
+  </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
   <PropertyGroup>
     <TargetExt>.pyd</TargetExt>
--- Python-2.7.18-orig/PCbuild/tcl.vcxproj	2025-12-12 20:03:35.208186900 +0100
+++ Python-2.7.18-mod/PCbuild/tcl.vcxproj	2025-12-12 21:55:18.009824100 +0100
@@ -64,6 +64,10 @@
     <OutDir>$(tcltkDir)</OutDir>
     <TargetPath>$(OutDir)bin\$(tclDLLName)</TargetPath>
   </PropertyGroup>
+  <!-- On ARM64, disable tcl as tcl/tk ARM64 support is not yet configured -->
+  <PropertyGroup Condition="'$(Platform)'=='ARM64'" Label="Configuration">
+    <ConfigurationType>Utility</ConfigurationType>
+  </PropertyGroup>
 
   <ItemGroup>
     <ExpectedOutputs Include="
--- Python-2.7.18-orig/PCbuild/tix.vcxproj	2025-12-12 20:03:35.244188500 +0100
+++ Python-2.7.18-mod/PCbuild/tix.vcxproj	2025-12-12 21:55:16.750871100 +0100
@@ -65,6 +65,10 @@
     <OutDir>$(tcltkDir)</OutDir>
     <TargetPath>$(tixDLLPath)</TargetPath>
   </PropertyGroup>
+  <!-- On ARM64, disable tix as tcl/tk ARM64 support is not yet configured -->
+  <PropertyGroup Condition="'$(Platform)'=='ARM64'" Label="Configuration">
+    <ConfigurationType>Utility</ConfigurationType>
+  </PropertyGroup>
 
   <ItemGroup>
     <ExpectedOutputs Include="$(TargetPath)" />
--- Python-2.7.18-orig/PCbuild/tk.vcxproj	2025-12-12 20:03:35.260187800 +0100
+++ Python-2.7.18-mod/PCbuild/tk.vcxproj	2025-12-12 21:55:19.517831800 +0100
@@ -65,6 +65,10 @@
     <OutDir>$(tcltkDir)</OutDir>
     <TargetPath>$(OutDir)bin\$(tkDLLName)</TargetPath>
   </PropertyGroup>
+  <!-- On ARM64, disable tk as tcl/tk ARM64 support is not yet configured -->
+  <PropertyGroup Condition="'$(Platform)'=='ARM64'" Label="Configuration">
+    <ConfigurationType>Utility</ConfigurationType>
+  </PropertyGroup>
 
   <ItemGroup>
     <ExpectedOutputs Include="
