--- setup.py.orig	2024-01-01 00:00:00.000000000 +0000
+++ setup.py	2024-01-01 00:00:00.000000000 +0000
@@ -30,7 +30,7 @@
 host_platform = get_platform()
 
 # Were we compiled --with-pydebug or with #define Py_DEBUG?
-COMPILED_WITH_PYDEBUG = ('--with-pydebug' in sysconfig.get_config_var("CONFIG_ARGS"))
+COMPILED_WITH_PYDEBUG = ('--with-pydebug' in (sysconfig.get_config_var("CONFIG_ARGS") or ''))
 
 # This global variable is used to hold the list of modules to be disabled.
 disabled_module_list = []
@@ -250,7 +250,7 @@
         global ARCH_RUN_32BIT
 
         if 'ARCHFLAGS' in os.environ and \
-            sysconfig.get_config_var("CONFIG_ARGS")):
+            (sysconfig.get_config_var("CONFIG_ARGS") or '')):
             # On Mac OS X we have historically provided an --enable-universalsdk
             # option which means '--enable-universalsdk=/' plus a bunch of magic
             # related to building extension modules properly.  This is typically
@@ -1303,7 +1303,7 @@
             # --with-pth=directory to configure causes this to use the
             # provided directory as the pth library.
             if '--with-pth' in sysconfig.get_config_var("CONFIG_ARGS"):
-                config_args = [arg for arg in sysconfig.get_config_var("CONFIG_ARGS").split()
+                config_args = [arg for arg in (sysconfig.get_config_var("CONFIG_ARGS") or '').split()
                                            if arg.startswith('--with-pth=')]
                 if config_args:
                     pth_dir = config_args[0][len('--with-pth='):]
@@ -1520,7 +1520,7 @@
 
         # More information on bundling with OS X:
         # http://mail.python.org/pipermail/python-dev/2012-November/122269.html
-        if '--with-system-expat' in sysconfig.get_config_var("CONFIG_ARGS"):
+        if '--with-system-expat' in (sysconfig.get_config_var("CONFIG_ARGS") or ''):
             expat_inc = []
             expat_lib = ['expat']
             expat_sources = []
@@ -1688,7 +1688,7 @@
         # ...except if explicitly disabled
         elif (platform in ('freebsd4', 'freebsd5', 'freebsd6', 'freebsd7', 'freebsd8')
             or platform.startswith("linux")) and \
-                sysconfig.get_config_var("CONFIG_ARGS")):
+                (sysconfig.get_config_var("CONFIG_ARGS") or '')):
             # Python was configured with --disable-ipv6
             return
 
@@ -2082,7 +2082,7 @@
             min_ffi_version = None
 
             if '--with-system-ffi' in sysconfig.get_config_var("CONFIG_ARGS"):
-                config_args = [arg for arg in sysconfig.get_config_var("CONFIG_ARGS").split()
+                config_args = [arg for arg in (sysconfig.get_config_var("CONFIG_ARGS") or '').split()
                                            if arg.startswith('--with-system-ffi=')]
                 if config_args:
                     ffi_inc = config_args[0][len('--with-system-ffi='):]
@@ -2160,7 +2160,7 @@
 
         # Determine if signalfd is supported
         use_signalfd = False
-        if not '--with-system-ffi' in sysconfig.get_config_var("CONFIG_ARGS"):
+        if not '--with-system-ffi' in (sysconfig.get_config_var("CONFIG_ARGS") or ''):
             use_signalfd = self.compiler.has_function('signalfd')
 
         ffi_inc = [sysconfig.get_config_var("LIBFFI_INCLUDEDIR")]
